tests/unittests/tests-printf_float/tests-printf_float.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  unittests
+ * @{
+ *
+ * @file
+ * @brief       Implementations of unit tests for the SAUL registry
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "embUnit/embUnit.h"
+
+#include "tests-printf_float.h"
+
+#define BUFSIZE             (10)
+
+static const float in0 = 2016.0349f;
+static const float in1 = 123.4567f;
+static const float in2 = 0.0f;
+
+static void sfprintf_float(void)
+{
+    char tmp[BUFSIZE];
+    char *str = tmp;
```

Comment:
I tried it and I got the following compilation error on every `TEST_ASSERT_EQUAL_STRING`:

```
In file included from /home/jgn/work/src/riot/sys/include/embUnit/embUnit.h:47:0,
                 from /home/jgn/work/src/riot/tests/unittests/tests-printf_float/tests-printf_float.c:24:
/home/jgn/work/src/riot/tests/unittests/tests-printf_float/tests-printf_float.c: In function ‘sfprintf_float’:
/home/jgn/work/src/riot/tests/unittests/tests-printf_float/tests-printf_float.c:39:43: error: invalid initializer
     TEST_ASSERT_EQUAL_STRING("2016.0349", str);
                                           ^
/home/jgn/work/src/riot/sys/include/embUnit/AssertImpl.h:50:44: note: in definition of macro ‘TEST_ASSERT_EQUAL_STRING’
         __typeof__(actual_) ____actual__ = actual_; \
                                            ^~~~~~~
```